.github/workflows/ci.yml

```diff
@@ -85,5 +85,12 @@ jobs:
           echo KRML_HOME=$(pwd)/karamel >> $GITHUB_ENV
       - name: build kyber
         run: |
+          NIX_SHELL="nix develop github:aeneasverif/charon/${{ github.sha }}#kyber-ci --command"
           cd libcrux-ml-kem
-          nix develop 'github:aeneasverif/charon/${{ github.sha }}#kyber-ci' --command ./c.sh
+          $NIX_SHELL ./c.sh
+          cd c
+          $NIX_SHELL cmake \
+            -DCMAKE_EXE_LINKER_FLAGS="-fuse-ld=mold" \
+            -DCMAKE_SHARED_LINKER_FLAGS="-fuse-ld=mold" \
+            -G "Ninja Multi-Config" -B build
+          $NIX_SHELL cmake --build build --config Release
```

Comment:
This will ebaluate nix things 3 times. Coule you write a script in `scripts/ci-build-kyber` and call it from here with a single `nix develop` invocation?

Reply:
I added it to the `kyber-ci` nix shell instead of putting it in `scripts/` though

Reply:
sure